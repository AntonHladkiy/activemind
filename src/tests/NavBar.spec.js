import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../components/NavBar';
let customUser={
    firstName:"",
    lastName:"",
    role:""
}
describe('NavBar logged in', () => {
    let wrapped = shallow(<NavBar loggedIn={true} user={customUser}></NavBar>);
    it('should render the NavBar Component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });
});
describe('NavBar not logged in', () => {
    let wrapped = shallow(<NavBar loggedIn={false} user={customUser}></NavBar>);
    it('should render the NavBar Component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });

});