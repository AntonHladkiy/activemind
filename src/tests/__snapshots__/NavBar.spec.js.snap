// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar logged in should render the NavBar Component correctly 1`] = `
<header
  className="d-flex flex-column p-3 px-md-4 mb-3 bg-white border-bottom shadow-sm"
>
  <nav
    className="navbar navbar-light bg-light justify-content-between"
  >
    <a
      className="navbar-brand"
    >
      Active Mind
    </a>
    <span
      className="navbar-text "
    >
      <span
        className="mr-2"
      >
        <Link
          to="/"
        >
           Main page
        </Link>
      </span>
      <span
        className="mr-2"
      >
        <Link
          to="/"
        >
           Log Out
        </Link>
      </span>
    </span>
  </nav>
</header>
`;

exports[`NavBar not logged in should render the NavBar Component correctly 1`] = `
<header
  className="d-flex flex-column p-3 px-md-4 mb-3 bg-white border-bottom shadow-sm"
>
  <nav
    className="navbar navbar-light bg-light justify-content-between"
  >
    <a
      className="navbar-brand"
    >
      Active Mind
    </a>
    <span
      className="navbar-text "
    >
      <span
        className="mr-2"
      >
        <Link
          to="/"
        >
           Main page
        </Link>
      </span>
      <span
        className="mr-2"
      >
        <Link
          to="/login"
        >
           Log In
        </Link>
      </span>
    </span>
  </nav>
</header>
`;
